source/reference/template-apps.txt

```diff
@@ -134,52 +134,52 @@ Template Apps Available
 
    * - Manage Database Views
      - ``triggers``
-     - Event-driven :ref:`Database Trigger <database-trigger>` template to update a view in a separate collection.
+     - Event-driven :ref:`Database Trigger <database-trigger>` template to update a view in a separate collection. Does *not* include a front-end app.
 
    * - Flutter + Realm Flutter SDK
      - ``flutter.todo.flex``
-     - Cross-platform application for Android, iOS, Windows and MacOS built with the :ref:`Realm Flutter SDK <flutter-intro>`.
+     - Cross-platform to-do list app for Android, iOS, Windows, MacOS, and Linux written in Dart using the :ref:`Realm Flutter SDK <flutter-intro>`.
 
-   * - React.JS + {+realm-web+} Boilerplate
+   * - React.JS + Realm Web SDK Boilerplate
      - ``web.mql.todo``
-     - Hosted Todo web app using the :ref:`{+realm-web+} SDK <web-intro>`.
+     - Hosted to-do list web app using the :ref:`Realm Web SDK <web-intro>`.
    
    * - GraphQL + React App Boilerplate
      - ``web.graphql.todo``
-     - Hosted Todo web app using the :ref:`GraphQL API <graphql-api>`
+     - Hosted to-do list app using the :ref:`GraphQL API <graphql-api>`.
 
-   * - Kotlin SDK + {+sync+} Starter (Flexible Sync)
+   * - Kotlin SDK + Atlas Device Sync Starter (Flexible Sync)
      - ``android.kotlin.todo.flex``
-     - Android app written in Kotlin using the :ref:`Realm Kotlin SDK <kotlin-intro>` to sync data to {+service-short+}'s server. Uses Flexible Sync.
+     - Android to-do list app written in Kotlin using the :ref:`Realm Kotlin SDK <kotlin-intro>` to sync data to App Services's server. Uses :ref:`Flexible Sync <flexible-sync>`.
```

Comment:
i noticed that throughout these descriptions it says "to sync data to App Service's server". 

i think this is an odd way of describing what Device Sync does. 

i think it could be better described as something like:

> Android to-to list app written in Kotlin using the Realm Kotlin SDK. Syncs local data to MongoDB Atlas using Device Sync.

this change should be applied to all the template apps that use Device Sync.

Reply:
updated for all applicable bullets